src/components/Planetarium/Planetarium.tsx

```diff
@@ -3,15 +3,34 @@ import { PlanetBio, AllData } from '../../interface';
 import './planetarium.css';
 import Planet from '../Planet/Planet';
 
-const Planetarium: React.FC<AllData> = ({ allPlanets }): JSX.Element => {
+const Planetarium: React.FC<AllData> = ({ allPlanets, sortKey }): JSX.Element => {
+
+  const addDescriptor = (data: any, key: any) => {
```

Comment:
could key be type string here? Does it need to be `key: any` and `data: any` ?

Reply:
fixed! Just trying to figure out the initial solution and forgot to double back.